beacon_node/beacon_chain/src/canonical_head.rs

```diff
@@ -597,19 +604,23 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         let mut fork_choice_write_lock = self.canonical_head.fork_choice_write_lock();
 
         // Recompute the current head via the fork choice algorithm.
-        fork_choice_write_lock.get_head(current_slot, &self.spec)?;
+        // TODO: Just with get_head, is it possible for finality to advance?
```

Comment:
I think yes, if there is unrealized finalization which gets actioned at an epoch boundary?